.pre-commit-config.yaml

```diff
@@ -94,8 +94,6 @@ repos:
         stages: [manual]
         additional_dependencies: &pyright_dependencies
         - pyright@1.1.258
--   repo: local
-    hooks:
```

Comment:
not relevant to this PR, but there's no need to keep repeating `repo: local`, so removing these extra ones while I'm here